src/ggrc/notifications/common.py

```
@@ -10,17 +10,33 @@
 """
 
 
+from flask import current_app
 from google.appengine.api import mail
+
 from ggrc import settings
-from flask import current_app
 
 
 def getAppEngineEmail():
+  """Return a valid email of an appengine user.
+
+  The email that get's returned should be used by appengine as a sender email.
+  """
   email = getattr(settings, 'APPENGINE_EMAIL')
   return email if mail.is_email_valid(email) else None
 
 
 def send_email(user_email, subject, body):
+  """Helper function for sending emails.
+
+  Args:
+    user_email (string): Email of the recepient.
+    subject (string): Email subject.
+    body (basestring): Html body of the email. it can contain unicode
+      characters and will be sent as a html mime type.
+
+  Returns:
+    True if the email was sent or False if something was wrong.
```

I've just noticed that we never check the return of this function. It might be better to have no return and to throw an exception on error, otherwise this can just silently fail.